docs/changelog.rst

```diff
@@ -1,5 +1,29 @@
+.. currentmodule:: fortnite_api
+
 .. _changelog:
 
 Changelog
 =========
-This page keeps track of all the changes to the project. For help on Migrating to Version 3 from Version 2, see the :ref:`Migration guide <migrating>`.
\ No newline at end of file
+
+.. _vp3p1p0:
+
+v3.1.0
+-------
+
+New Features
+~~~~~~~~~~~~~
+- Add :attr:`fortnite_api.MaterialInstance.product_tag`
+- Add :attr:`fortnite_api.ShopEntry.colors`
+- Add :attr:`fortnite_api.ShopEntry.offer_tag`
+- Add :attr:`fortnite_api.ShopEntryLayout.rank`
+- Add :attr:`fortnite_api.ShopEntryNewDisplayAsset.render_images`
+- Create new object :class:`fortnite_api.ShopEntryRenderImage`
```

Comment:
This conflicts with #26. I recommend removing these lines here, merging this PR, and leaving it up to #26 to update this changelog with its changes.
```suggestion
```